operations/network/management_network/hardware/ex2500.md

```diff
@@ -0,0 +1,51 @@
+# EX2500 installation and cabling
+
+This documentation helps network administrators and support personnel install and manage EX2500 cabinet in shasta management network.
+
+### HPE Cray EX2500 in quick summary:
+
+A single rack can accommodate up to 24 compute blade slots within 3 compute chassis.
+
+One rack supports the following:
+* Up to 3 compute chassis
+* Up to 3 power shelves with 4 rectifiers per shelf - 12 total 15kw rectifiers per rack. (Note: 1 rectifier per shelf is used for redundancy)
+* 3 PDUs (1 per power shelf)
+* 3 power input whips (3-phase)
+* Maximum of 24 compute blades
+* Maximum of 24 HPE Slingshot switch blades
+* Up to 2 in-rack Coolant distribution units for a max of 141KVA cooling capacity
+
+#### How does EX2500 connect to Shasta management network:
+
+Typically EX2500 cabinet connects in Shasta management network to the CDU switches (sw-cdu-00x).
+
+However in some smaller setups where CDU switches are not present such as "test and development", it is possible to connect the EX2500 cabinet to sw-leaf-bmc switch.
+
+#### How to connect EX2500 if CDU switches are not present:
+
+To connect EX2500 cabinet to a smaller system with out CDU switches will require a custom switch configuration from the network administrator.
```

Comment:
```suggestion
To connect an EX2500 cabinet to a smaller system without CDU switches will require a custom switch configuration from the network administrator as this layout is currently not supported by [CANU](https://github.com/cray-hpe/canu), the CSM management network configuration generation tool.  Please note that [custom switch configs](https://github.com/Cray-HPE/canu/blob/main/docs/network_configuration_and_upgrade/custom_config.md) should still be processed through [CANU](https://github.com/cray-hpe/canu) so that CANU does not overwrite them when generating switch configs.
```